test/recipes/30-test_evp_data/evppbe.txt

```diff
@@ -38,14 +38,14 @@ r = 8
 p = 1
 Key = 7023bdcb3afd7348461c06cd81fd38ebfda8fbba904f8e3ea9b543f6545da1f2d5432955613f0fcf62d49705242a9af9e61e85dc0d651e40dfcf017b45575887
 
-# NB: this test requires more than 1GB of memory to run so it will hit the
+# NB: this test requires more than 2GB of memory to run so it will hit the
 # scrypt memory limit and return an error. To run this test without error
 # uncomment out the "maxmem" line and comment out the "Result"
 # line
```

Comment:
please read this comment

Reply:
Is this a suggestion to revert this test case?

Reply:
Yes - unless you have a known answer for the modified test

Reply:
Out of curiosity, I commented just the line `Result = SCRYPT_ERROR` below,
and it looks like the unmodified test does already exceed the memory bound.
How can this be the case here and not in the other test?

Reply:
???

The error returned didn't indicate the memory limit being hit.

Reply:
I'll dig in tomorrow.

Reply:
I've confirmed your findings.  Via the PBE path, the default maximum memory use is different to going directly.
I'll leave the PBE test case alone (it's already been reverted).  The KDF test case does need the change.